src/ArtistList.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import ArtistDetail from './ArtistDetail';
```

Comment:
Rename the imported variable to camelCase for the same reason as listed above. The exported entity is not a class here. It's an array. Use PascalCase only while exporting Classes or React components.